### DISCUSSION TITLE
Logging Support

### DISCUSSION THREAD

**Discussion by jmann-idt** [dasniko/testcontainers-keycloak#77] [Q&A] [answered]
I am trying to understand what the preferred way of setting custom logging levels would be when using this container.  While I see that one can use `withEnv("KC_LOG_LEVEL", "DEBUG")` to see the root logging level, I am not sure how I would set custom logging levels.
At the moment, I am able to so this by overriding `setCommand` and adding additional command parts such as `--log-level=INFO,my.category:DEBUG`.  That said, I see there are explicit comments that such additions are not supported and I assume there is a reason why.

Is this something that verges into an enhancement request, maybe not arbitrary command parts, but some helper methods specifically for logging?  Is this capability already supported and I am not aware of how to take advantage of it?

Thanks for your thoughts!

**Comment by dasniko** [comment, marked as answer]
Hi @jmann-idt 
Thanks for coming up with this.
Have you already tried to set `withEnv("KC_LOG_LEVEL", "INFO,my.category:DEBUG")`?
This should actually work.

**Reply by jmann-idt** [reply]
Yes that does work!  Thanks so much for the insight!
